index.js

```diff
@@ -980,6 +980,42 @@
   //. ```
   S.init = S.slice(0, -1);
 
+  //# take :: Number -> [a] -> Maybe [a]
+  //.
+  //. Returns Just a list containing the first n elements of the given
+  //. list or Just the first n characters of a given string. If n exceeds
+  //. the length of the list or string, returns Nothing.
```

Comment:
If we mention `slice` we should mention that negative indexes are not supported: even though they are nonsensical you can guarantee someone will try to use it at some point.